guides/deployment/gitlab-ci.md

```diff
@@ -1 +1,31 @@
+---
+tags: [guide]
+---
+# GitLab CI
 
+## Step 1: Add `gitlab-ci.yml` pipeline
+
+```yml
+image: node:lts-buster # https://hub.docker.com/_/node
+
+before_script:
+  - npm install --global retypeapp # https://retype.com/guides/getting-started/
```

Comment:
Suggestion: can you get your environment with .NET 5 SDK installed? So you could install retype with `dotnet tool install --global retype`.

This could make the install much faster and smaller download.

Reply:
Maybe using `5.0-buster-slim-amd64` see https://hub.docker.com/_/microsoft-dotnet-aspnet

Reply:
Hey there, im using Gitlab CI/CD with Yarn + Dockerfile (img) and the pipeline takes about 27 seconds, should be fast enough i guess :P 

Pipeline Status:
![image](https://user-images.githubusercontent.com/6567620/188343596-d69703b6-d090-4691-84c7-9506ae84bdba.png)

Pipeline Commands:
![image](https://user-images.githubusercontent.com/6567620/188343703-eff4406c-cbac-4829-bb50-d5340cbe31cc.png)